tests/atomspace/AtomTableUTest.cxxtest

```diff
@@ -276,27 +275,4 @@ public:
         TS_ASSERT(table->getHandlex("28675194", MY_CONCEPT_NODE) != Handle::UNDEFINED);
         TS_ASSERT(table->getHandle(MY_INHERITANCE_LINK, os) != Handle::UNDEFINED);
     }
-
-    void testUnaddedCustomAtomTypes()
-    {
-        Handle hv1 = Handle(createNode(VARIABLE_NODE, "$1"));
-        Handle hv2 = Handle(createNode(VARIABLE_NODE, "$2"));
-        HandleSeq hvs{hv1, hv2};
-        Handle hlist = Handle(createVariableList(hvs));
-
-        VariableListPtr v_ptr(VariableListCast(hlist));
-        TS_ASSERT_EQUALS(v_ptr->get_variables().varset.count(hv1), 1);
-        TS_ASSERT_EQUALS(v_ptr->get_variables().varset.count(hv2), 1);
-
-        Handle hlist_added = table->add(hlist, false);
-
-        VariableListPtr v_added_ptr(VariableListCast(hlist_added));
-        TS_ASSERT_EQUALS(v_added_ptr->get_variables().varset.count(hv1), 0);
-        TS_ASSERT_EQUALS(v_added_ptr->get_variables().varset.count(hv2), 0);
-
-        Handle hv1_added = table->add(hv1, false);
-        Handle hv2_added = table->add(hv2, false);
-        TS_ASSERT_EQUALS(v_added_ptr->get_variables().varset.count(hv1_added), 1);
-        TS_ASSERT_EQUALS(v_added_ptr->get_variables().varset.count(hv2_added), 1);
-    }
```

Comment:
Perhaps I was hasty; you can test something like this, but the test above seemed to be enshrining weird bugs and undefined behavior.  You really should discard the pointers hv1, hv2 as soon as possible; createVariableList is not exactly a "public" api anyway.  

Anyway, atoms-specific tests should go in the /tests/atoms directory.